run.sh

```diff
@@ -6,8 +6,9 @@ set -o pipefail
 
 DIRECTORY="${1}"
 CURRENT_BRANCH=$(git branch --show-current)
-export PREFIX=${2}
-export NEW_VERSION="${GITHUB_HEAD_REF#$PREFIX}"
```

Comment:
removed `export` from here, these variables shouldnt be used outside of this script